customer-stack/create-resources.py

```diff
@@ -219,48 +224,48 @@ def create_describemodel_ef(snowflake_cursor, api_integration_name, api_gateway_
     logger.info("Creating External function: AWS_AUTOPILOT_DESCRIBE_MODEL [api_integration_name=%s, api_gateway_url=%s]", api_integration_name, api_gateway_url)
 
     describemodel_request_translator_str = ("""create or replace function %s(EVENT OBJECT)
-        returns OBJECT LANGUAGE JAVASCRIPT AS 
-        $$ 
-        let item = EVENT.body.data[0][1]; 
-        let payload = { 
-            \"AutoMLJobName\" : item + \"-job\" 
-        }; 
+        returns OBJECT LANGUAGE JAVASCRIPT AS
+        $$
+        let item = EVENT.body.data[0][1];
+        let payload = {
+            \"AutoMLJobName\" : item + \"-job\"
+        };
```

Comment:
There is a lot of purely formatting changes in this file which make it hard to review. Would it be possible to restore original formatting and only add those (couple of ) lines which are effectively being changed?

Reply:
Yes, there were lot of unnecessary spaces at lots of places. To keep the code in a good state. The actual code changes are here: https://github.com/aws-samples/amazon-sagemaker-integration-with-snowflake/blob/vpc-mode/customer-stack/create-resources.py#L193-L202 . Having correctly formatted code will help upcoming changes to the repo.